setup.json

```diff
@@ -86,7 +86,7 @@
             "PyCifRW~=4.4",
             "ase~=3.18",
             "matplotlib~=3.3,>=3.3.4",
-            "pymatgen>=2019.7.2,<=2022.02.03,!=2019.9.7",
```

Comment:
Was this limit here for any reason?

Reply:
There was, they had introduced some backwards compatible changes, but we already accounted for them apparently. Anyway the upper limit was wrong (the version doesn't even exist) and is no longer necessary

Reply:
yeh I think they yanked some pip releases